### DISCUSSION TITLE
Sziasztok egy kicsit el akadtam egy projektemel tunda e valaki segiteni ?

### DISCUSSION THREAD

**Discussion by Erik-is21** [OHF-Voice/intents#2378] [Supported Sentences]
Egy salus márkáju termékele játszom , egy pár napja .
De nem tudom pontosan hogy müködik itt a home-assistant ba .
Ha valki tudna nekem adni egy DC (Discord) Linket elérhetöséget , aki tudna nekem ebben segiteni azt meg kösszöném !  
Pár alkatrész kod ( GGPC1 , RX10RF , UGE600 , és van egy termosztátom )

**Comment by tetele** [comment]
This repo is dedicated to voice commands for Home Assistant's built-in voice assistant. To get support with HA integrations, use what you feel are the most appropriate method from the ones [listed here](https://www.home-assistant.io/help/).
